runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -44,8 +44,6 @@
 import org.corfudb.protocols.service.CorfuProtocolMessage.EpochCheck;
 import org.corfudb.protocols.wireprotocol.ClientHandshakeHandler;
 import org.corfudb.protocols.wireprotocol.ClientHandshakeHandler.ClientHandshakeEvent;
```

Comment:
nit: import for NonNull above could be removed.

Reply:
Done.